datasets/tr/fcib/crawler.py

```diff
@@ -54,6 +54,18 @@ def split(text: Optional[str]) -> List[str]:
     return [s.strip() for s in REGEX_SPLIT.split(text)]
 
 
+def parse_passport_numbers(pass_no: Optional[str]) -> List[str]:
+    if pass_no is None:
+        return []
+    # split only with value ##-## or ##/##
+    multi_pass_no_reg = re.compile(r"(^\d+[-]\d+$)|(^\d+/\d+(/\d+)?$)")
```

Comment:
let's compile this once and use as a constant. Maybe we can enforce some number of digit constraints